web/app.go

```diff
@@ -42,27 +45,61 @@ type Dependencies struct {
 	subscriptionsService services.SubscriptionsService
 	hostsService         services.HostsService
 	sapSystemsService    services.SAPSystemsService
+	tagsService          services.TagsService
 }
 
 func DefaultDependencies() Dependencies {
 	consulClient, _ := consul.DefaultClient()
 	engine := gin.Default()
 	store := cookie.NewStore([]byte("secret"))
 
-	araService := ara.NewAraService(araAddrDefault)
+	db, err := InitDB()
+	if err != nil {
+		log.Fatalf("failed to connect database: %s", err)
```

Comment:
I see a logged message only here. shouldn't we panic the application? Or enter in a connection tries loop?

Reply:
P.S.: we wrote down a tech-debt about retry/backoff mechanisms, as for now exiting is good enough